src/app/client/src/app/modules/groups/services/groups/groups.service.ts

```diff
@@ -373,4 +373,32 @@ getActivity(groupId, activity, mergeGroup, leafNodesCount?) {
   getDashletData(courseHeirarchyData, aggData) {
     return this.groupCservice.activityService.getDataForDashlets(courseHeirarchyData, aggData);
   }
+
+  // Temporary fix to get telemetry contexrt for group exception logs 
+
+  /**
+   * generate the telemetry context data by using router details
+   */
+  getTelemetryContext() {
```

Comment:
Too much of code, that to lot fo assumptions specific to activity & group. 
not happy with the logic being used even though refactoring has to be done for this.